src/components/common/PeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -346,12 +347,14 @@ export default class PeopleAndRoles extends Mixins(CommonMixin, DateMixin, OrgPe
    * Sets state properties to add an org/person.
    * @param roles The roles of this item.
    * @param type The incorporator (party) type of this item.
+   * @param actions The actions of this item.
    */
-  private initAdd (roles: RoleIF[], type: PartyTypes): void {
+  private initAdd (roles: RoleIF[], type: PartyTypes, actions = [ActionTypes.ADDED]): void {
```

Comment:
This allows us to specify actions, or be compatible with the old code if we don't specify them.